website/docs/concepts/router.md

```diff
@@ -45,7 +45,7 @@ matched, the router navigates to the path with `not_found` attribute. If no rout
 a message is logged to console stating that no route was matched.
```

Comment:
Overall im missing documentation about availability to set basename and what that does